src/components/multiselectContainer.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, createElement } from "react";
+
+interface WrapperProps {
+    class: string;
+    mxObject?: mendix.lib.MxObject;
+    mxform: mxui.lib.form._FormBase;
+    style: string;
+    readOnly: boolean;
+    friendlyId: string;
+}
+
+export interface ContainerProps extends WrapperProps {
+    datetimeformat: string;
+    datePattern: string;
+    timePattern: string;
+    decimalPrecision: string;
+    groupDigits: string;
+    onChangeMicroflow: string;
+    onChangeMicroflowShowProgress: string;
+    onChangeMicroflowProgressMessage: string;
+    retrieveType: string;
+    dataAssociation: string;
+    displayTemplate: string;
+    variableName: string;
+    variableAttribute: string;
+    _variableContainer: string;
+    dataConstraint: string;
+    sortContainer: string;
+    sortOrder: string;
+    sortAttribute: string;
+    retrieveMicroflow: string;
+    reloadDataViaAttribute: string;
+    showLabel: string;
+    fieldCaption: string;
+    formOrientation: string;
+    labelWidth: string;
+    addSelectAll: string;
+    addFilter: string;
+    caseSensitiveFilter: string;
+    itemsToDisplay: string;
+    disabled: string;
+    disabledViaAttribute: string;
+    visible: string;
+    visibleViaAttribute: string;
+    selectAllText: string;
+    noneSelectedText: string;
+    allSelectedText: string;
+    numberSelectedText: string;
+}
+
+// export interface ContainerState {
+//  }
+
+export default class MultiselectSetSelectorContainer extends Component<ContainerProps> {
+
+    readonly state = {
+        datetimeformat: undefined
+    };
+
+    constructor(props: ContainerProps) {
+        super(props);
+
+        // this.handleChange = this.handleChange.bind(this);
+    }
+
+    render() {
+
+        return createElement("div",
```

Comment:
What exactly are you rendering here? Please check with the mx widget already how its rendering...